dbt_subprojects/dex/models/aggregator_trades/dex_aggregator_base_trades.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+    schema ='dex_aggregator'
+    , alias = 'base_trades'
+    , partition_by = ['block_month', 'blockchain', 'project']
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address']
+    , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{% set base_trade_models = [
+    ref('lifi_base_trades')
+] %}
+
+with base_union as (
+    SELECT *
+    FROM
+    (
+        {% for model in base_trade_models %}
+        SELECT
+            blockchain
+            , project
+            , version
+            , block_date
+            , block_month
+            , block_time
+            , token_bought_symbol
+            , token_sold_symbol
+            , token_pair
+            , token_bought_amount
+            , token_sold_amount
```

Comment:
I will try to remove these 5 columns from base_trades in order to remove `erc20` source table from base_trades spells.